### PR TITLE
Update  Documentation 

### LINKED ISSUE #67: [DOC]: improve the documentation
Any contribution to the documentation (typos, formatting, more clarity...) is welcome!

### PR DESCRIPTION
**Pull request by kadatatlukishore**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include 
relevant motivation and context. List any dependencies that are required for this 
change.

Fixes #67

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not 
  work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions 
so that the tests can be reproduced. Please also list any relevant details for your 
test configuration

**Test Configuration**:

* OS: Windows 10
* Python version: 3.8

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -71,28 +71,30 @@ The following third-party packages are required:
...
 
 Download & Installation
 =======================
 
 BCDI is available from:
- * Python Package Index: pip install bcdi
- * https://github.com/carnisj  for the most updated version
- * upgrade existing version from GitHub: pip install --upgrade git+https://github.com/carnisj/bcdi.git
+ * Python Package Index: ``pip install bcdi``
+ * `Updated Version <https://github.com/carnisj/>`_
```
Comment (carnisj) [inline README.rst:81]: Could you change this to:

```suggestion
* `Most updated version on GitHub <https://github.com/carnisj/>`_
```

File: HISTORY.rst [hunk 2]
```diff
@@ -1,26 +1,26 @@
 Version 0.1.2
 -------------
 
-* Refactor: remove circular imports from modules
+* Refactor: ``remove circular imports from modules``
```
Comment (carnisj) [inline HISTORY.rst:4]: Please revert this to:
```suggestion
* Refactor: remove circular imports from modules
```

File: HISTORY.rst [hunk 3]
```diff
@@ -276,6 +276,6 @@ Version 0.0.2
 
 Version 0.0.1
 -------------
-* Initial add, for the moment only the main scripts have been converted and checked:`` strain.py`` and ``prepare_cdi_mask.py``
+* Initial add, for the moment only the main scripts have been converted and checked:``strain.py`` and ``prepare_cdi_mask.py``
```
Comment (carnisj) [inline HISTORY.rst:279]: Please add a space
```suggestion
* Initial add, for the moment only the main scripts have been converted and checked: ``strain.py`` and ``prepare_cdi_mask.py``
```

File: README.rst [hunk 4]
```diff
@@ -92,7 +92,7 @@ Changelog
 .. include:: ../HISTORY.rst
   :end-before: Version 0.1.0
 
-See the full .. doc:`Changelog<changelog>`
+See the full  :doc:`Changelog<changelog>`
```
Comment (carnisj) [inline README.rst:95]: Honestly here I am not sure what the effect is, let see.

**Review by carnisj — CHANGES_REQUESTED** [review]
Hi, thank you for the nice work. I added few suggestions (also so that we can practice the whole PR workflow).

**Comment by kadatatlukishore** [comment]
Hello @carnisj , Made the required changes. Please review

**Review by carnisj — APPROVED** [review]

**Pull request merged**